### PR TITLE
Fixed input bugs.

### PR DESCRIPTION
**Pull request by Ranray**

### CODE REVIEW

**Review by BeeHiveJava — APPROVED** [review]

File: Soccar [UI]/src/nl/soccar/ui/input/Keyboard.java [hunk 1]
```diff
@@ -58,6 +53,9 @@
         HANDBRAKE = new ArrayList<>();
         HANDBRAKE.add(KeyCode.SPACE);
     }
+    
+    // Stores the keys that are being pressed at any time.
+    private final List<KeyCode> PRESSED_KEYS = new ArrayList<>();
```
Comment (TehMessiah) [inline Soccar [UI]/src/nl/soccar/ui/input/Keyboard.java]: MINOR Rename this field "PRESSED_KEYS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS00116)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: Soccar [UI]/src/nl/soccar/ui/input/Keyboard.java (hunk 1 above)
Comment (BeeHiveJava) [inline Soccar [UI]/src/nl/soccar/ui/input/Keyboard.java]: PRESSED_KEYS -> pressedKeys.

**Comment by BeeHiveJava** [comment]
Niet mergen s.v.p.

**Pull request merged**